lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,47 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+
+  function registerLiveTail(params, callbacks, overrideRequestId){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    knownComputations[requestId] = {
+      onMessage: callbacks.onMessage,
+      transport: transports.WEBSOCKET,
+      requestId: requestId,
+    };
+    knownComputations[requestId].streamController = getLiveTailStreamController();
+
+    activeSocket.send(JSON.stringify({
+      type: 'registerlivetail',
+      channel: requestId,
+      query: params.query,
+      throttleOptions: params.throttleOptions,
+    }));
+  }
+
+  function stopLiveTail(requestId){
+    activeSocket.send(JSON.stringify({
```

Comment:
You should check if `requestId` maps to an known computation; you can reuse `removeComputation()` to help:

```js
function stopLiveTail(requestId) {
  if (removeComputation(requestId)) {
    activeSocket.send(JSON.stringify({ ... });
  }
}
```

Reply:
done